weblate/addons/forms.py

```diff
@@ -46,6 +46,27 @@ def save(self):
         return self._addon.instance
 
 
+class GenerateMoForm(BaseAddonForm):
+    path = forms.CharField(
+        label=_('Path of generated MO file'),
```

Comment:
Some help what is expected to be set here (and what blank value means) could be useful.

Reply:
I shall see what I can set.